### DISCUSSION TITLE
how i can set config.yaml for a model if my dataset  have a train and test folder??

### DISCUSSION THREAD

**Discussion by BrunoMartelli01** [open-edge-platform/anomalib#1484] [Q&A]
there is some tag to put in the config.yaml file? beacuse i only found normal_dir and abnormal_dir

**Comment by willyfh** [comment]
hi @BrunoMartelli01 , simply speaking, we can think that the `normal_dir` is actually your `train` folder. Because the training will use the data from the `normal_dir`. And the `abnormal_dir` is your `test` folder, since the abnormal images are only used to evaluate the model.

**Comment by samet-akcay** [comment]
Maybe this could help?
https://anomalib.readthedocs.io/en/latest/markdown/guides/how_to/data/custom_data.html
